### PR TITLE
[cert-manager-1.15] CM-359 : Update operator CSV as required for the konflux

### PR DESCRIPTION
**Pull request by bharath-b-rh**
Changes are required for building images using konflux to comply with enterprise policy contract.

### CODE REVIEW

File: .snyk [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# References:
+# https://docs.snyk.io/scan-applications/snyk-code/using-snyk-code-from-the-cli/excluding-directories-and-files-from-the-snyk-code-cli-test
+# https://docs.snyk.io/snyk-cli/commands/ignore
+exclude:
+  global:
+    - vendor/**
```
Comment (TrilokGeer) [inline .snyk:6]: Is it better to scan through the projects vendor for vulnerabilities unless it is made sure the dependent packages are scanned independently.

**Comment by swghosh** [comment]
/lgtm
/label docs-approved
/label px-approved 
this is intended in pulling the same changes in master from #209 into cm-1.15 
/retitle [cert-manager-1.15] CM-359 : Update operator CSV as required for the konflux
/label tide/merge-method-rebase

**Pull request merged**